### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mousumi098**
### What do you want to achieve?
-

### PR Checklist

- [ ] I have tested it locally and all functionalities are working fine.
- [ ] I have compared it with mocks and all design elements are the same.
- [ ] I have tested it in IE Browser.

### How was the code tested?

-

### What new thing you came across while writing this code? 
-

### In case you fixed a bug then please describe the root cause of it? 
-

NOTE: Make sure you're comparing your branch with the correct base branch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,8 @@ It lets you easily add real-time live chat and in-app messaging in your websites
 
 Kommunicate has a powerful human + chatbot hybrid platform, where you can integrate Dialogflow and Amazon Lex chatbot without using any code. You can also integrate any third-party or custom chatbot in Kommunciate as well. The automatic bot to human handoff enabled, customizable and rich-messaging equipped chat-widget gives you ample options to make your chatbot more powerful. 
 
-![Chat-widget](https://user-images.githubusercontent.com/32258004/95323958-9dd83c00-08bc-11eb-8744-bbf33140e00d.png)
+![Chat-widget](https://user-images.githubusercontent.com/91717856/215320134-f4f7308c-0e31-45cd-8c5b-619c4bba63b8.png)
```
Comment (devashishmamgain) [inline README.md:13]: image is not good, it is showing restart thing, is that intentional?

File: README.md [hunk 2]
```diff
@@ -20,13 +21,14 @@ Kommunicate has a powerful human + chatbot hybrid platform, where you can integr
 
 [Kompose](https://dashboard.kommunicate.io/bots/bot-builder) is a Kommunicate’s bot builder that help you in building your own bot, a techie, non-techie, or a person who doesn’t have any idea about chatbots can also build the bot. Anyone can create a chatbot with the Kompose without any assistance.
 
-![Kompose-build](https://user-images.githubusercontent.com/38066371/87525821-a519f000-c6a7-11ea-90d6-97e8fab4d1b3.gif)
+![Kompose-build](https://user-images.githubusercontent.com/91717856/215320213-8f09b877-dc17-46b8-aac2-1caf4a0b5be4.gif)
```
Comment (devashishmamgain) [inline README.md:24]: quality is not good, also please don't use terms like "Test 1", "Test 2". put some real project examples

**Review by devashishmamgain — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mousumi098) [reply README.md:13]: No no let me check once

File: README.md (hunk 2 above)
Comment (mousumi098) [reply README.md:24]: Okay will create new one and add it and will check the quality also

**Comment by Sanjeevsinghnegi** [comment]
@mousumi098 please check this PR

**Pull request closed without merging**